ow_faults/src/FaultInjector.cpp

```diff
@@ -26,8 +30,14 @@ FaultInjector::FaultInjector(ros::NodeHandle node_handle)
   m_dist_pitch_ft_sensor_pub = node_handle.advertise<geometry_msgs::WrenchStamped>(ft_sensor_dist_pitch_str, 10);
 
   //power fault publishers and subs
-  m_fault_power_state_of_charge_pub = node_handle.advertise<std_msgs::Float64>("temporary/power_fault/state_of_charge", 10);
-  m_fault_power_temp_pub = node_handle.advertise<std_msgs::Float64>("temporary/power_fault/temp_increase", 10);
+  m_power_soc_sub = node_handle.subscribe("/power_system_node/state_of_charge", 
+                                          1000, 
+                                          &FaultInjector::powerSOCListener, 
+                                          this); 
+  m_power_temperature_sub = node_handle.subscribe("/power_system_node/battery_temperature", 
+                                                  1000, 
+                                                  &FaultInjector::powerTemperatureListener, 
+                                                  this); 
```

Comment:
Maybe it doesn't matter too much but in general I recommend that you move the subscriptions to be after you advertise the topics.